### ISSUE TITLE
Confluence metrics

### ISSUE THREAD

**Issue by AuspeXeu** [simplesteph/kafka-stack-docker-compose#64]
Would be great if there was an env var that allows us to disable the metrics being sent to confluence :)

**Comment by simplesteph** [comment]
There’s one but I can’t remember which. Look up in the confluent documentation and I’m happy to take a PR

**Comment by AuspeXeu** [comment]
Yeah, I had a look already but couldn't find anything. They at least don't seem to expose it as an ENV var.

**Comment by simplesteph** [comment]
@AuspeXeu try adding this environment variable:
```
KAFKA_CONFLUENT_SUPPORT_METRICS_ENABLE=false
```
This should result of it being disabled.

`KAFKA_FOO_BAR=BAZ` results in a new property added to Kafka: `foo.bar=baz`
